src/app/globals.css

```
@@ -46,6 +46,38 @@
   50%       { opacity: 0.55; }
 }
 
+@keyframes slideFromRight {
```

Stylelint (17.12.0)

[error] Expected keyframe name "slideFromRight" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)